code/game/objects/items/defib.dm

```diff
@@ -192,7 +193,10 @@
 		A.UpdateButtonIcon()
 
 /obj/item/defibrillator/proc/make_paddles()
```

Comment:
this proc is ABSOLUTELY USELESS.

on the defib
```DM
var/paddle_type = /your/fav/type/here
```

wherever makepaddles is used
```DM
var/blah = make_paddles() >>>>>> var/blah = new paddle_type(src)
```

Reply:
i don't know what these words mean cobby man i have not touched this part at all in the pr

Reply:
pls don't make me fix someone else's shitcode i can barely handle my own